docs/guides/testing.md

```diff
@@ -17,7 +17,7 @@ Before jumping into test suites we will wrap the `createTestContext` with a patt
 
 ```ts
 // tests/__helpers.ts
-import { createTestContext, TestContext } from 'nexus/testing'
+import { createTestContext as originalCreateTestContext, TestContext } from 'nexus/testing'
```

Comment:
Looks like this was renamed but isn't used below. `createTestContext()` recursively calls itself instead of calling `originalCreateTestContext`